### ISSUE TITLE
Easy decrypt using key pair A and encrypt using key pair B

### ISSUE THREAD

**Issue by gdubicki** [voxpupuli/hiera-eyaml#251]
We have cases where we would like to automatically re-encrypt a bunch of files using a different key pair.

It would be nice if we could have a new command, let's say 'rekey, which could be internally similar to what `hiera edit` does, but which would with single command, in a non-interactive way:

1. open a file using key pair A and decrypt encrypted values there,
2. encrypt values using key pair B and write to the file again.

We are thinking about creating a PR for this, but perhaps this may be doable with some existing cli spells or someone has this ready in some fork, so I would like to report this need here first. :)

**Comment by TJM** [comment]
One of the options on the command line is `recrypt`. It feels like that would be the right place to add that functionality. (honestly, I thought that was what that was for, but it doesn't appear to be)

**Comment by FransUrbo** [comment]
Yeah, this is starting to be extremely important to me as well! `recrypt` did seem to be the obvious choice, but not sure exactly what that do. Same as `encrypt` it seems?!

**Comment by FransUrbo** [comment]
@gdubicki Are you guys still thinking about a PR to fix this?
